Stripe/STPCard.m

```diff
@@ -135,6 +148,35 @@ - (BOOL)isEqualToCard:(STPCard *)other {
     return [self.cardId isEqualToString:other.cardId];
 }
 
+- (NSString *)description {
```

Comment:
Ah ok. I thought the default NSObject implementation of `description` would just forward to `debugDescription` or something, but doing whatever gets proper NSLog out I think is good.